.terraform-version

```diff
@@ -0,0 +1 @@
+0.12.24
```

Comment:
I think we should remove this. This is useful in principle, but since we constantly want to test with newer terraform versions for compatibility, this actually hinders our ability in practice.